### DISCUSSION TITLE
Json Serializer not Working

### DISCUSSION THREAD

**Discussion by AXDevTimothy** [abuzuhri/Amazon-SP-API-CSharp#226] [Q&A]
I'm a little confused here.  At first, My request body was being serialized twice (within the RequestService class).  Once by JsonConvert.SerializeObject, and once again then by restSharp Request.addJsonBody().  I commented out the var json = JsonConvert.SerializeObject and it serialized fine then.  However, Enums do not serialize to string, and null properties are not excluded as they should be.  It sounds like I need to serialize using Json.Net rather then allowing restSharp to serialize, however the Request.JsonSerializer = new JsonNetSerializer() is commented out and I dont see a reference to JsonNetSerializer in the project.  I'm surprised this is working for others. Am I missing something?  Thanks in advance.
![image](https://user-images.githubusercontent.com/106777343/176286801-adf7ea08-91f7-440d-9d8e-cd133251f602.png)

**Comment by abuzuhri** [comment]
Hi @AXDevTimothy 

I'm not using **JsonNetSerializer**  any more the project working perfect with many developer , what your problem exactly ?
